src/main/java/com/atlassian/db/replica/internal/aurora/AuroraConnectionSupplier.java

```diff
@@ -0,0 +1,9 @@
+package com.atlassian.db.replica.internal.aurora;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+@FunctionalInterface
+public interface AuroraConnectionSupplier {
+    Connection get() throws SQLException;
```

Comment:
We should specify replica here (it's important to connect to specific replica)

Reply:
we get connection per replica on a higher level, this is a secondary interface (outdated anyway)